### PR TITLE
add AIRBYTE_ENTRYPOINT for kubernetes support

### PR DESCRIPTION
**Pull request by jrhizor**
resolves most of the checks from https://github.com/airbytehq/airbyte/issues/3903 :

* set AIRBYTE_ENTRYPOINT in base image
* set AIRBYTE_ENTRYPOINT for connectors with custom entrypoints
* set AIRBYTE_ENTRYPOINT for normalization
* bump/publish all image versions

I'll publish tonight or tomorrow morning

### CODE REVIEW

**Comment by jrhizor** [comment]
Can't wait for https://github.com/airbytehq/airbyte/issues/1866 btw

**Review by sherifnada — APPROVED** [review]
can you also update templates? Checklist for updating templates is in the PR checklist template

File: airbyte-integrations/connector-templates/source-generic/Dockerfile [hunk 1]
```diff
@@ -1,7 +1,8 @@
 FROM scratch
 
 ## TODO Add your dockerfile instructions here
-
+## TODO uncomment the below line
```
Comment (davinchia) [inline airbyte-integrations/connector-templates/source-generic/Dockerfile:4]: good call @sherifnada !

**Review by davinchia — APPROVED** [review]
:shipit:

**Comment by jrhizor** [comment]
/publish connector=connectors/source-google-search-console-singer

**Comment by jrhizor** [comment]
/publish connector=connectors/source-google-workspace-admin-reports

**Comment by jrhizor** [comment]
/publish connector=connectors/source-stock-ticker-api-tutorial

**Comment by jrhizor** [comment]
/publish connector=connectors/source-amazon-seller-partner false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-google-workspace-admin-reports false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-googleanalytics-singer false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-zendesk-support-singer false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-amazon-seller-partner run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-google-workspace-admin-reports run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-googleanalytics-singer run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-quickbooks-singer run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-recharge run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-salesforce-singer run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-zendesk-support-singer run-tests=false

**Comment by jrhizor** [comment]
/publish connector=connectors/source-google-sheets run-tests=false

**Comment by jrhizor** [comment]
Status of all of the connectors: https://docs.google.com/document/d/1ad6n6yhC9P4V7gyTB2G6lwpL5jVTWZj69s8xJDcWuxQ/edit

If the tests pass here I'm going to merge.

**Pull request merged**